python/pyspark/ml/tests.py

```diff
@@ -499,6 +499,24 @@ def test_linear_regression(self):
         except OSError:
             pass
 
+    def test_logistic_regression(self):
+        lr = LogisticRegression(maxIter=1)
+        path = tempfile.mkdtemp()
+        lr_path = path + "/lr"
```

Comment:
I didn't catch this before: Could you please call this "/logreg" instead of "/lr" so it's a different name from the LinearRegression test?